pkg/features/features.go

```diff
@@ -101,4 +101,9 @@ const (
 	// owner: @mvladev
 	// alpha: v1.23.0
 	AdminKubeconfigRequest featuregate.Feature = "AdminKubeconfigRequest"
+
+	// UseDNSRecords enables using DNSRecords resources for Gardener DNS records instead of DNSProvider and DNSEntry resources.
+	// owner: @stoyanr
+	// alpha: v1.26.0
```

Comment:
Will most likely not be in `v1.26`
```suggestion
	// alpha: v1.27.0
```

Reply:
Changed to v.1.27.0 as proposed.